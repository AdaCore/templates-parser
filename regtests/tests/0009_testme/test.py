from test_support import *

run('testme', ["testme15.tmplt"])
run('print_tree', ["testme15.tmplt"])
