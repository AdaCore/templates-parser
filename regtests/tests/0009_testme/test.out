
  here

[INFO] testme15.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[IF] (@_TWO_@ > 0)
  [TEXT]   here
[END_IF]
